crates/conductor_lib/src/conductor/base.rs

```diff
@@ -437,6 +450,22 @@ impl Conductor {
             .expect("Must be able to spawn thread")
     }
 
+    pub fn spawn_trace_reporter_thread(&mut self) -> thread::JoinHandle<()> {
+        let reporters = self.trace_reporters.clone();
+        thread::Builder::new()
+            .name("signal_multiplexer".to_string())
```

Comment:
do we want a whole thread here? could be a lighter weight futures task - then use a futures channel allowing a more efficient `.await` rather than thread::sleep - just a thought.

if this stays a thread maybe mention jaeger or tracing in the thread name.